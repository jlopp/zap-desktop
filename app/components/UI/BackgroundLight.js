import React from 'react'
import { Box } from 'rebass'

/**
 * @render react
 * @name BackgroundLight
 * @example
 * <BackgroundLight />
 */
class BackgroundLight extends React.Component {
  render() {
    return <Box bg="lightBackground" color="white" {...this.props} />
  }
}

export default BackgroundLight
