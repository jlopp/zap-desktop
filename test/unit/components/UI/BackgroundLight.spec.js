import React from 'react'
import BackgroundLight from 'components/UI/BackgroundLight'
import renderer from 'react-test-renderer'

describe('component.UI.BackgroundLight', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<BackgroundLight />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
