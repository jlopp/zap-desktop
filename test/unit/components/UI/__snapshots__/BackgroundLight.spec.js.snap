// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component.UI.BackgroundLight should render correctly 1`] = `
.c0 {
  color: white;
  background-color: lightBackground;
}

<div
  className="c0"
  color="white"
/>
`;
